crates/amalthea/src/comm/ui_comm.rs

```diff
@@ -466,6 +466,12 @@ pub enum UiFrontendEvent {
 	#[serde(rename = "show_html_file")]
 	ShowHtmlFile(ShowHtmlFileParams),
 
+	/// This event is used to signal that the stored messages the front-end
+	/// replays when constructing multi-output plots should be reset. This
+	/// happens for things like a holoviews extension being changed.
+	#[serde(rename = "clear_webview_preloads")]
+	ClearWebviewPreloads,
```

Comment:
Is there some reason this is getting added in now alongside this?

Reply:
Maybe this was from a previous update to the comm definition used to generate code, and it showed up now?
https://github.com/posit-dev/positron/blame/a0e53f8b220ebdcc815cf144ea141313954924d9/positron/comms/ui-frontend-openrpc.json#L450